README.md

```diff
@@ -11,18 +11,22 @@ Intended for [gettext](https://www.npmjs.com/search?q=gettext) or similar transl
 This library provides custom [Tagged Template Literals](https://developer.mozilla.org/en/docs/Web/JavaScript/Reference/Template_literals#Tagged_template_literals) interpolation functions.
 
 ```javascript
-tag = (template, ...substitutions) => ...
+const tag = (template, ...substitutions) => ...
+const substitution = 10;
 
 tag`Some statement that can use ${substitution}`
+// => Some statement that can use 10
 ```
 
-Template Literals allow **substitutions**. These will appear as wildcards in the untranslated text.
+Template Literals allow **substitutions**. These will appear as **tokens** in the untranslated text.
 
 ```
 msgid "Some statement that can use ____"
```

Comment:
This cannot be done by default. Read the comment that follows.

Maybe I should make that more obvious.